spec/requests/api/person_escort_records_controller_create_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::PersonEscortRecordsController do
+  describe 'POST /person_escort_records' do
+    include_context 'with supplier with spoofed access token'
+
+    let(:response_json) { JSON.parse(response.body) }
+    let(:profile) { create(:profile) }
+    let(:profile_id) { profile.id }
+    let(:framework) { create(:framework, framework_questions: [build(:framework_question)]) }
+    let(:framework_version) { framework.version }
+
+    let(:person_escort_record_params) do
+      {
+        data: {
+          "type": 'person_escort_records',
+          "attributes": {
+            "version": framework_version,
+          },
+          "relationships": {
+            "profile": {
+              "data": {
+                "id": profile_id,
+                "type": 'profiles',
+              },
+            },
+          },
+        },
+      }
+    end
+
+    before do
+      post '/api/v1/person_escort_records', params: person_escort_record_params, headers: headers, as: :json
+    end
+
+    context 'when successful' do
+      let(:application) { create(:application, owner: supplier) }
+      let(:access_token) { create(:access_token, application: application).token }
```

Comment:
Is there a need to use a real access token here? If a spoofed token will work then that's preferable for faster tests. Generally we only need to use a real token if authorisation depends on the supplier access to models. Swapping to a spoofed token also means no need to create the application or supplier records.